content/blog/2021-03-08-week-1.md

```diff
@@ -3,27 +3,27 @@ title: 'Week 1 in review'
+Btw, I installed [Insomnia](https://insomnia.rest/) as an open source alternative to Postman for testing endpoints. (Well, the client is open source, their paid hosted services are not.) It's pretty nice, I would recommend it. 
 
-Yulia uses the [jwt_sessions gem](https://github.com/tuwukee/jwt_sessions) which keeps track of tokens with the help of Redis, so I setup Redis, too. That might be useful for caching and other stuff in the future, too.
+In the blog post, Yulia uses the [jwt_sessions gem](https://github.com/tuwukee/jwt_sessions) which keeps track of tokens with the help of Redis, so I setup Redis, too. That might be useful for caching and other stuff in the future, too.
 
-I set up CI with GitHub Actions, which was smoother than expected.
+I set up continuous integration (CI) with GitHub Actions, which was smoother than expected.
 After that I installed [Rubocop](https://rubocop.org/) for linting (I get very sloppy if there are no checks in place, and the initial fixing I already had to do was quite surprising...) and added it as a step in CI as well. Boom!
 Integration with [Coveralls](https://coveralls.io) for code coverage and [Sentry](https://sentry.io) for exception tracking are still on my list though.
 
-While setting up automation of things I decided to setup up CD to a staging environment on a hobby dyno on Heroku as well. I want the production app to be deployed on a VPS via docker (-compose) but it will take some time until that is all done and available. So, why not have an always up-to-date version running for easy testing and have an alternative way to deploying? Exactly!
+While setting up automation of things I decided to setup up continuous delivery (CD) to a staging environment on a hobby dyno on Heroku as well. I want the production app to be deployed on a VPS via docker (-compose) but it will take some time until that is all done and available. So, why not have an always up-to-date version running for easy testing and have an alternative way to deploying? Exactly!
 
 For documentation and a developer blog I set up this page, using [Hugo](https://gohugo.io/) as static site generator and the [Hugo Whisper Theme](https://github.com/zerostaticthemes/hugo-whisper-theme) and made some adjustments. Github Actions also helps here with automatic deployment to GiHub Pages.
 
-Too much GitHub? Probably, but it definitely helps getting setup quickly. I'm more familiar with the interface than with GitLab's plus I don't want the hassle to maintain (and pay for) my own infrastructure at this point.
+Too much GitHub? Probably, but it definitely helps getting setup quickly. I'm more familiar with the interface than with GitLab's, plus I don't want the hassle to maintain (and pay for) my own infrastructure at this point.
```

Comment:
I think most readers won't know what you mean by "too much Github". Either remove or clarify this.